confirmation/migrations/0008_confirmation_validity_in_days.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.16 on 2020-09-28 00:13
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('confirmation', '0007_add_indexes'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='confirmation',
+            name='validity_in_days',
+            field=models.PositiveIntegerField(default=10),
+        ),
```

Comment:
This migration needs a `RunPython` to set the original `validity_in_days` to match what would have been set were these created using the code in `models.py`.

Also @mateuszmandera do you think an `expiration_date` field would be cleaner than the `validity_in_days` data model?  I guess the benefit of the integer number of days is that it makes for cleaner edit UI, at the cost of less expressive power and less convenience in writing raw SQL queries to find unexpired objects.

Reply:
Yeah, `expiration_date` seems cleaner - but I'm not sure why this would affect the edit UI? The edit UI can still express validity in days

Reply:
@mateuszmandera I think the benefit of having date/time field (`expiration_date`) is that to enable users to pick a specific date and time for link expiration.